src/SecTester.Scan/DefaultScanFactory.cs

```diff
@@ -0,0 +1,98 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using SecTester.Core;
+using SecTester.Core.Utils;
+using SecTester.Scan.Models;
+using SecTester.Scan.Target.Har;
+using Response = SecTester.Scan.Target.Har.Response;
+
+namespace SecTester.Scan;
+
+public class DefaultScanFactory : ScanFactory
+{
+  private const int HarDefaultNameMaxHostLength = 200;
```

Comment:
Har default name max host length? What? 
```suggestion
  private const int MaxSlugLength = 200;
```